packages/mui-material/src/Rating/Rating.js

```diff
@@ -464,7 +479,6 @@ const Rating = React.forwardRef(function Rating(inProps, ref) {
     if (isFocusVisibleRef.current === false) {
       setFocusVisible(false);
     }
-
     const newFocus = -1;
```

Comment:
Revert the above change

```suggestion

    const newFocus = -1;
```